src/content/contributing/adding-desci-projects/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Adding DeSci projects
+description: The policy we use when adding a links to projects on the DeAci page on ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding projects {#adding-projects}
+
+We want to make sure we show a variety of projects and give a good snapshot of the DeSci landscape.
+
+Anyone is free to suggest a project to list on the DeSci page on ethereum.org. Equally, anyone who notices a project that is no longer relevant or no longer meets our eligibility criteria, they are free to suggest we remove it.
+
+## The decision framework {#the-decision-framework}
+
+### Criteria for inclusion: the must-haves {#the-must-haves}
+
+- **Open source code/data** - Openness of code and data is a core DeSci principle, so DeSci projects must not be closed source. The codebase should be accessible and ideally open to PRs.
+- **DeSci projects should be demonstrably decentralized** - This include being governed by a DAO, or by building with decentralized tech stack including non-custodial wallets. It probably involves auditable smart contracts at publicly-known Ethereum addresses.
```

Comment:
> DeSci projects should be demonstrably decentralized

Agree in principle. Might be a bit vauge/subjective.

> publicly-known Ethereum addresses

phrasing here is weird. I interpreted it as 'well-known Ethereum addresses'.

Reply:
```suggestion
- **DeSci projects should be demonstrably decentralized** - This could include being governed by a DAO, or by building with a decentralized tech stack including non-custodial wallets. It probably involves auditable smart contracts on Ethereum.
```